testing/tests/index.test.js

```diff
@@ -1016,6 +1016,24 @@ test("bcd table extraction followed by h3", () => {
   expect(doc.body[4].value.isH3).toBeTruthy();
 });
 
+test("specification table and bcd extraction", () => {
+  const builtFolder = path.join(
+    buildRoot,
+    "en-us",
+    "docs",
+    "web",
+    "spec_table_extraction"
+  );
+  expect(fs.existsSync(builtFolder)).toBeTruthy();
+  const jsonFile = path.join(builtFolder, "index.json");
+  const { doc } = JSON.parse(fs.readFileSync(jsonFile));
+  expect(doc.body[0].type).toBe("prose");
+  expect(doc.body[1].type).toBe("specifications");
```

Comment:
```suggestion
  expect(doc.body[1].type).toBe("specifications");
  // Let's look a bit deeper into these kinds of sections
  expect(doc.body[1].value.foo).toBe(bar);
  ...
```

Reply:
I wonder, whether it would even make sense to unpack `doc.body` into separate variables (to save some lookups and have a speaking name).